tests/regression/test_infinite_loop.cpp

```diff
@@ -120,6 +123,8 @@ main(void)
 
         std::cout << "OK" << std::endl;
 
+        std::this_thread::sleep_for(std::chrono::milliseconds(1200));
+

```

Comment:
Not quite sure what is the goal of this but it's no biggie. Is it for avoiding crash by exiting too fast?

Reply:
It's to make the test more reliable. I've noticed while tuning function attributes, that the test would crash on the Apple CI but not x86-64 CI. The reason of crash was that kernel function contained unreachable inst, but the the x86-64 managed to exit (with success) before the kernel function was ran by the CPU driver, while Apple did actually run the kernel and crashed.